tests/test_core/test_sequence.py

```diff
@@ -2173,6 +2173,22 @@ def test_relative_position_step_GT_one(integer_seq):
     assert got == 4
 
 
+@pytest.mark.parametrize("sliced", (False, True))
+@pytest.mark.parametrize("rev", (False, True))
```

Comment:
might be good to also test that when `sliced=False` that the underlying seq is not modified

Reply:
done